tests/unit/s2n_psk_test.c

```diff
@@ -516,6 +516,69 @@ int main(int argc, char **argv)
 
             EXPECT_SUCCESS(s2n_connection_free(conn));
         }
+
+        /* Test s2n_connection_set_psks */
```

Comment:
Maybe also test what happens if the length of the list is too long / too short / null? This is an API, so the inputs could be very wrong